### PR TITLE
Docs: Add note to CONTRIBUTING to prevent people wasting time on autofix

### PR DESCRIPTION
**Pull request by ForbesLindesay**

### CODE REVIEW

**Comment by ForbesLindesay** [comment]
That line is very definitely intended to be emphasised, not to be a header.  If it is an acceptable solution, I'll disable MD036 in the Makefile.js?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Please be sure to read the contribution guidelines before making or requesting a change.
 
+**N.B. We are not accepting new autofixes at this time.  We may begin accepting autofixes for new rules in 3.0.0.**
```
Comment (hzoo) [inline CONTRIBUTING.md]: I think autofixes would be accepted for whitespace rules?

**Comment by alberto** [comment]
Rather that pointing specifically that out, we should probably link to the docs (is it really documented somewhere?) that you should open an issue if there is none, and wait for it to be accepted, before starting to work on it.

**Comment by ForbesLindesay** [comment]
@hzoo, I can update the notice if that's the case I guess.  What do you want to do about the markdown validator?

@alberto I don't think that's what's at issue here.  It is a waste of time for people to even open that issue.

**Comment by ilyavolodin** [comment]
@ForbesLindesay We will still accept autofixes for whitespace rules (as long as they don't conflict with others too badly). Could you also prefix the whole paragraph with **Important:** instead of making it all bold? Bolded Important should be sufficient to attract attention.

**Comment by nzakas** [comment]
I'm sorry, I don't think this is a good solution. If we add anything, it should be somewhere in the formal contributor guide: http://eslint.org/docs/developer-guide/contributing/. My hunch is that "requesting a change" might be the best location.

@ForbesLindesay I'm sorry you wasted some time. This is part of why we require issues for all code pull requests, it gives us an opportunity to stop someone from writing something we know we can't use.

**Comment by ilyavolodin** [comment]
Closing as it doesn't look like something we want to merge. Sorry to have wasted your time:-(

**Pull request closed without merging**